app.js

```diff
@@ -103,26 +101,47 @@ for (var j = 0; j < storeHours.length; j++) {
   }
   var hourEl = document.createElement('td');
   hourEl.textContent = hourlyTotal;
-  rowEl.appendChild(hourEl);
+  footEl.appendChild(hourEl);
 }
 
 document.body.appendChild(tableEl);
 
+var storeFormEl = document.getElementById('new-store-form');
+
+storeFormEl.addEventListener('submit', handleSubmit);
+
+function handleSubmit(event){
+  event.preventDefault();
+  event.stopPropagation();
+
+  var storeName = event.target.storeName.value;
+  var minCust = parseInt(event.target.minCust.value);
+  var maxCust = parseInt(event.target.maxCust.value);
+  var avgCookies = parseFloat(event.target.avgCookies.value);
+  console.log('User pressed submit button on form');
+  console.log(minCust, maxCust);
+
+  var store = new CookieStore(storeName, minCust, maxCust, avgCookies);
+  console.log(store);
+
+  var rowEl = document.createElement('tr');
```

Comment:
This logic between 127 and 145 if why you need to have a render method on your prototype.  If you had a method, you could do all of this just by calling that method instead of having to rewrite all of this logic in your event listener